<launch>
  <node pkg="trajectory_planning" type="trajectory_planner" name="trajectory_planner" output="screen"/>
</launch>
